openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -918,22 +918,22 @@ These checks may be performed in any order.
 
 Credential Response can be immediate or deferred. The Credential Issuer MAY be able to immediately issue a requested Credential and send it to the Client.
 
-In other cases, the Credential Issuer MAY NOT be able to immediately issue a requested Credential and would want to send a `transaction_id` parameter to the Client to be used later to receive a Credential when it is ready. The HTTP status code MUST be 202 (section 10.2.3 of [@!RFC2616]).
+In other cases, the Credential Issuer MAY NOT be able to immediately issue a requested Credential and would want to send a `transaction_id` parameter to the Client to be used later to receive a Credential when it is ready. The HTTP status code MUST be 202 (see Section 10.2.3 of [@!RFC2616]).
```

Comment:
RFC2616 has been obsoleted (twice!) - this https://datatracker.ietf.org/doc/html/rfc9110#section-15.3.3 is the current section for [202 Accepted](https://datatracker.ietf.org/doc/html/rfc9110#name-202-accepted)

```suggestion
In other cases, the Credential Issuer MAY NOT be able to immediately issue a requested Credential and would want to send a `transaction_id` parameter to the Client to be used later to receive a Credential when it is ready. The HTTP status code MUST be 202 (see Section 15.3.3 of [@!RFC9110]).
```